tests/modules/nf-core/ilastik/multicut/nextflow.config

```diff
@@ -0,0 +1,18 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: "ILASTIK_MULTICUT" {
+        ext.args = '--output_filename_format=multicut_segmentation.tiff'
```

Comment:
This should be in the main script (since this handles a file, all arguments in `ext.args` need to be optional, non-file arguments)

Reply:
Actually I believe its optional. You can leave this argument out and ilastik will still write and output .tiff mask, I am not quite sure how it determines the filename if left empty though...

Reply:
You can add this to the main script as an argument (`--output_filename_format=${prefix}.tiff`). This way the output file name can be adjusted according to the input by the user